libs/wire-api/src/Wire/API/Team/Member.hs

```diff
@@ -195,10 +199,33 @@ type HardTruncationLimit = (2000 :: Nat)
 hardTruncationLimit :: Integral a => a
 hardTruncationLimit = fromIntegral $ natVal (Proxy @HardTruncationLimit)
 
+-- | Like 'ListType', but without backwards-compatible and boolean-blind json serialization.
+data NewListType
```

Comment:
Roundtrip test missing, I think

Reply:
Hah, I think I was half-consciously hoping you'd miss that.  :)  My excuse is that it's included in the EJPD* roundtrip tests, but I'll add it anyway to protect against refactorings.